app/config/config_dev.yml

```diff
@@ -42,6 +42,6 @@ swiftmailer:
     transport: smtp
     host: 'localhost'
     port: 1025
-    username: null
-    password: null
+    username: "%switftmailer_username%"
+    password: "%switftmailer_password%"
```

Comment:
You can just remove then.
The default values for username & password are `null`.

```php
->scalarNode('username')->defaultNull()->end()
->scalarNode('password')->defaultNull()->end()
```